resources/eventing/charts/nats/values.yaml

```diff
@@ -171,22 +271,48 @@ statefulSetPodLabels:
 # Annotations to add to the NATS Service
 serviceAnnotations: {}
 
+# additionalContainers are the sidecar containers to add to the NATS StatefulSet
+additionalContainers: []
+
+# additionalVolumes are the additional volumes to add to the NATS StatefulSet
+additionalVolumes: []
+
+# additionalVolumeMounts are the additional volume mounts to add to the nats-server and nats-server-config-reloader containers
+additionalVolumeMounts: []
+
```

Comment:
same, remove these additionalContainers, additionalVolumes and additionalVolumeMounts, if not used.